src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0 8px;
+  display: block;
+}
+
+.big-form,
+.small-form {
+  margin-top: 20px;
+}
+
+.big-input,
+.small-input {
+  width: 100%;
+
+  font-family: "Avenir-Book", sans-serif;
```

Comment:
add font-family for html and here you can use `inherit`
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit